pymc_marketing/clv/plotting.py

```diff
@@ -332,33 +332,26 @@ def plot_probability_alive_matrix(
         max_recency=max_recency,
     )
     # FIXME: This is a hotfix for ParetoNBDModel, as it has a different API from BetaGeoModel
-    #  We should harmonize them!
+    transaction_data = pd.DataFrame(
+        {
+            "customer_id": np.arange(mesh_recency.size),  # placeholder
+            "frequency": mesh_frequency.ravel(),
+            "recency": mesh_recency.ravel(),
+            "T": max_recency,
+        }
+    )
     if isinstance(model, ParetoNBDModel):
-        transaction_data = pd.DataFrame(
-            {
-                "customer_id": np.arange(mesh_recency.size),  # placeholder
-                "frequency": mesh_frequency.ravel(),
-                "recency": mesh_recency.ravel(),
-                "T": max_recency,
-            }
-        )
-
         Z = (
             model.expected_probability_alive(
                 data=transaction_data,
-                future_t=0,  # TODO: This can be a function parameter in the case of ParetoNBDModel
+                future_t=0,  # TODO: This if block can be removed if
```

Comment:
Can be removed if what XD ?

Reply:
I suppose there are worse things to rush through than TODO comments 😆